tests/test_model.py

```diff
@@ -1144,60 +1159,181 @@ def test_with_complaint(self):
+        field data from provider, metadata wrangler, admin interface, etc. editions.
+        """
+        # create different types of editions, all with the same identifier
+        edition_admin = self._edition(data_source_name=DataSource.LIBRARY_STAFF, with_license_pool=False)
+        edition_mw = self._edition(data_source_name=DataSource.METADATA_WRANGLER, with_license_pool=False)
+        edition_od, pool = self._edition(data_source_name=DataSource.OVERDRIVE, with_license_pool=True)
+ 
+        edition_mw.primary_identifier = pool.identifier
+        edition_admin.primary_identifier = pool.identifier
+
+        # set overlapping fields on editions
+        edition_od.title = u"OverdriveTitle1"
+        [joe], ignore = Contributor.lookup(self._db, u"Sloppy, Joe")
+        joe.family_name, joe.display_name = joe.default_names()
+        edition_od.add_contributor(joe, Contributor.AUTHOR_ROLE)
+
+        edition_mw.title = u"MetadataWranglerTitle1"
+        edition_mw.subtitle = u"MetadataWranglerSubTitle1"
+        [bob], ignore = Contributor.lookup(self._db, u"Bitshifter, Bob")
+        bob.family_name, bob.display_name = bob.default_names()
+        edition_mw.add_contributor(bob, Contributor.AUTHOR_ROLE)
+
+        edition_admin.title = u"AdminInterfaceTitle1"
+        [jane], ignore = Contributor.lookup(self._db, u"Doe, Jane")
+        jane.family_name, jane.display_name = jane.default_names()
+        edition_admin.add_contributor(jane, Contributor.AUTHOR_ROLE)
+
+        pool.set_presentation_edition(None)
+
+        edition_composite = pool.presentation_edition
+
+        assert_not_equal(edition_mw, edition_od)
+        assert_not_equal(edition_od, edition_admin)
+        assert_not_equal(edition_admin, edition_composite)
+        assert_not_equal(edition_od, edition_composite)
+
+        # make sure admin pool data had precedence
+        eq_(edition_composite.title, u"AdminInterfaceTitle1")
+
+        # make sure data not present in the higher-precedence editions didn't overwrite the lower-precedented editions' fields
+        eq_(edition_composite.subtitle, u"MetadataWranglerSubTitle1")
+        license_pool = edition_composite.is_presentation_for
+        eq_(license_pool, pool)
+        
+        # is_primary_for_work can only happen if the pool has a work associated with it.
+        # add a work, and re-call set_presentation_edition, and test the primarity.
+        work = self._work()
+        # default testing work object creates and sets a primary edition, and we want an clean slate
+        work.primary_edition = None
+
+        # we haven't set a primary yet
+        eq_(edition_mw.is_primary_for_work, False)
+        eq_(edition_od.is_primary_for_work, False)
+        eq_(edition_admin.is_primary_for_work, False)
+        eq_(edition_composite.is_primary_for_work, False)
+
+        work.license_pools.append(pool)
+        pool.set_presentation_edition(None)
```

Comment:
Set default policy value to `None` so it can effectively be ignored until policy is completely removed. :)